src/app/select-trajectory/select-trajectory.page.ts

```diff
@@ -34,6 +35,60 @@ export class SelectTrajectoryPage implements OnInit {
     public locationService: LocationService
   ) {}
 
+  private CLICK_INTERVAL = 500
+  private DEBUG_WINDOW_CLICKS = 8
+  private lastOnClicks = []
+
+  // fired on title click
+  // used for multiple click detection
+  private async onTitleClick() {
```

Comment:
functions that are called from the template can't be private (but angular only complains in production builds)